### PR TITLE
Add tests for the autocorrect attribute.

### PR DESCRIPTION
**Pull request by whsieh**
See: <https://github.com/whatwg/html/issues/3595>.

### CODE REVIEW

**Review by annevk — APPROVED** [review]
From a cursory read these look accurate and comprehensive, thanks for writing them!

File: html/editing/editing-0/autocorrection/autocorrection.html [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+    e.autocorrect = false;
+    assert_false(e.autocorrect);
+    assert_equals(e.getAttribute('autocorrect'), 'off');
+
+    e.autocorrect = 0;
+    assert_false(e.autocorrect);
+    assert_equals(e.getAttribute('autocorrect'), 'off');
+  });
+}, "Test setting the autocorrect boolean attribute.");
```
Comment (annevk) [inline html/editing/editing-0/autocorrection/autocorrection.html:48]: Nit: you want "IDL attribute" here as boolean attribute is a term of art in HTML that means something else.

**Comment by josepharhar** [comment]
In "checks" it says that safari is passing 8 of the 9 tests. Is this expected?

**Comment by annevk** [comment]
Yes: https://bugs.webkit.org/show_bug.cgi?id=246344.

**Pull request merged**